src/test/iv_client.c

```diff
@@ -406,6 +406,39 @@ progress_function(void *data)
 	return NULL;
 }
 
+static struct crt_proto_rpc_format my_proto_rpc_fmt_iv[] = {
+	{
+		.prf_flags	= 0,
+		.prf_req_fmt	= &CQF_RPC_TEST_FETCH_IV,
+		.prf_hdlr	= NULL,
+		.prf_co_ops	= NULL,
+	}, {
+		.prf_flags	= 0,
+		.prf_req_fmt	= &CQF_RPC_TEST_UPDATE_IV, 
+		.prf_hdlr	= NULL,
+		.prf_co_ops	= NULL,
+	}, {
+		.prf_flags	= 0,
+		.prf_req_fmt	= &CQF_RPC_TEST_INVALIDATE_IV, 
+		.prf_hdlr	= NULL,
+		.prf_co_ops	= NULL,
+	}, {
+		.prf_flags	= 0,
+		.prf_req_fmt	= &CQF_RPC_SHUTDOWN,
+		.prf_hdlr	= NULL,
+		.prf_co_ops	= NULL,
+	}
+};
+
+static struct crt_proto_format my_proto_fmt_iv = {
+        .cpf_name = "my-proto-iv",
+        .cpf_ver = 0,
```

Comment:
We will want to change these version numbers over time so a \#define would be appropriate here, and you then need to pass the same value into CRT_PROTO_OPC() later.